File: backend/src/main/java/rs/raf/student/model/Destination.java
package rs.raf.student.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Destination {

    private Long id;

    private String name;

    private String description;

}

File: frontend/src/types/destination.ts
export type Destination = {
  id:          number,
  name:        string,
  description: string,
}

export type DestinationCreate = {
  name:        string,
  description: string,
}

export type DestinationUpdate = {
  id:           number,
  name?:        string,
  description?: string,
}
